operator/main.go

```diff
@@ -223,11 +227,17 @@ func runOperator(cmd *cobra.Command) {
 		startSynchronizingCiliumNodes()
 	}
 
-	if synchronizeServices {
-		startSynchronizingServices()
-	}
+	if kvstoreEnabled() {
+		if synchronizeServices {
+			startSynchronizingServices()
+		}
+
+		if synchronizeNodes {
```

Comment:
Is there any concern behind moving starting of the the `NodeWatcher` before we have initialized the kvstore below? Changes to bootstrap ordering always catch my eye.

Reply:
I did not find any, otherwise, I wouldn't have done this change.

Reply:
Noticed that unlike the service code, the node init code does not run sync

Reply:
Moved the code to after kvstore.Setup()